cnxauthoring/tests/test_views.py

```diff
@@ -72,6 +72,54 @@ def storage_cls(self):
         from ..storage.main import BaseStorage
         return BaseStorage
 
+    def test_update_content_state_with_parsing(self):
```

Comment:
I still need to write a utility test. Also, I might change the names of some variables like this one.